corehq/blobs/migrate.py

```diff
@@ -356,7 +356,7 @@ def get_type_code(doc):
         self.get_type_code = get_type_code
 
     def migrate(self, filename=None, reset=False, max_retry=2, chunk_size=100, **kw):
-        doc_provider = self.get_document_provider(**kw)
+        doc_provider = self.get_document_provider(date_range=kw.pop('date_range', None))
```

Comment:
This will not work with the default implementation of `get_document_provider`

https://github.com/dimagi/commcare-hq/blob/04a58ddb7fc79b8d7a17f81a742b81a62f6685d6/corehq/blobs/migrate.py#L398

That's why I [suggested only passing `date_range` if exists in `kw`](https://github.com/dimagi/commcare-hq/pull/24342/files/ff4c7d47a6ca1dd51eb2241d50c62c46225bac5b#r285127097).

Reply:
Of course, updated.